website/src/components/ConsoleTitle.tsx

```diff
@@ -0,0 +1,101 @@
+import { Terminal } from 'lucide-react';
+import { useState, useEffect } from 'react';
+
+export default function ConsoleTitle() {
+    const fullText = "RazorConsole";
+    const [text, setText] = useState("");
+    const [isDeleting, setIsDeleting] = useState(false);
+    const [isPaused, setIsPaused] = useState(false);
+
+    useEffect(() => {
```

Comment:
Have you considered using a third party library for typing animations, like typed.js?